common/components/register_model/spec.yaml

```diff
@@ -5,7 +5,7 @@ type: command
 
 is_deterministic: True
 
-display_name: Register model
+display_name: Register model (Public Preview)
```

Comment:
@ManojBableshwar, is this the approach you want? I don't see any Preview tags in this PR.

Reply:
@ManojBableshwar can you please share what preview tags to add?
Should it be like:

`preview`: `True`

Reply:
Added tags:

`Preview: ""`